lib/menu.rb

```diff
@@ -0,0 +1,12 @@
+class Menu
+  attr_reader :list
+  
+  def initialize
+    @list = [
```

Comment:
Your Menu Class looks great!
- In the future if you wanted to add an item your menu, this class allows you to add that functionality whilst also following the SRP (Single Responsibility Principle)

Potential improvements:
- Might be better to keep individual 'meals' in a 'meal' class.
   - the 'meal' would be used to represent a single meal
   - it could have the 'name' and 'price' as attributes
   - if you needed to change the price or the name of the meal in future this is perfect place to do that as the Meal Class is the one describing any meal you create.